docs/source/index.rst

```diff
@@ -100,6 +100,7 @@ Topics in this section will help you get started with ExecuTorch.
    :caption: Backends
    :hidden:
 
+   backends-overview
```

Comment:
Swap the position of "using-executorch-faqs" and "using-executorch-building-from-source" above